### PR TITLE
Release `13.0` commit and back to development mode

### PR DESCRIPTION
**Pull request by frouioui**
## Description

This pull request pushes the `v13.0.0` release commit and the "back to dev" mode commit too.

### CODE REVIEW

**Review by mattlord — APPROVED** [review]

File: examples/operator/operator.yaml [hunk 1]
```diff
@@ -5432,7 +5432,7 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: vitess-operator
-          image: planetscale/vitess-operator:latest
+          image: planetscale/vitess-operator:v2.6.0
```
Comment (deepthi) [inline examples/operator/operator.yaml:5435]: Same question here.

**Comment by deepthi** [comment]
This looks like a legitimate failure
```
--- FAIL: TestGen4SelectDBA (0.01s)
[246](https://github.com/vitessio/vitess/runs/5289711853?check_suite_focus=true#step:7:246)
panic: runtime error: index out of range [2] with length 2 [recovered]
[247](https://github.com/vitessio/vitess/runs/5289711853?check_suite_focus=true#step:7:247)
	panic: runtime error: index out of range [2] with length 2
[248](https://github.com/vitessio/vitess/runs/5289711853?check_suite_focus=true#step:7:248)
```

File: examples/operator/101_initial_cluster.yaml [hunk 2]
```diff
@@ -8,12 +8,12 @@ metadata:
...
 spec:
   images:
-    vtctld: vitess/lite:v13.0.0-rc1
-    vtgate: vitess/lite:v13.0.0-rc1
-    vttablet: vitess/lite:v13.0.0-rc1
-    vtbackup: vitess/lite:v13.0.0-rc1
+    vtctld: vitess/lite:v13.0.0
+    vtgate: vitess/lite:v13.0.0
+    vttablet: vitess/lite:v13.0.0
+    vtbackup: vitess/lite:v13.0.0
```
[the remark this answers is not in the document]
Comment (mattlord) [reply examples/operator/101_initial_cluster.yaml:14]: What are you referring to?

[the remark this answers is not in the document]
Comment (mattlord) [reply examples/operator/101_initial_cluster.yaml:14]: If you're asking about the container image with the tag, yes: https://hub.docker.com/r/vitess/lite/tags

**Comment by frouioui** [comment]
> This looks like a legitimate failure
> 
> ```
> --- FAIL: TestGen4SelectDBA (0.01s)
> [246](https://github.com/vitessio/vitess/runs/5289711853?check_suite_focus=true#step:7:246)
> panic: runtime error: index out of range [2] with length 2 [recovered]
> [247](https://github.com/vitessio/vitess/runs/5289711853?check_suite_focus=true#step:7:247)
> 	panic: runtime error: index out of range [2] with length 2
> [248](https://github.com/vitessio/vitess/runs/5289711853?check_suite_focus=true#step:7:248)
> ```

This test is flaky++, it was fixed on `main` by @vmg

File: examples/operator/operator.yaml (hunk 1 above)
Comment (mattlord) [reply examples/operator/operator.yaml:5435]: No, not yet: https://hub.docker.com/r/planetscale/vitess-operator/tags

We're working on that here: https://github.com/planetscale/vitess-operator/pull/253

We have to make a number of API related changes in the operator for Vitess 13.0 (calls moved form wrangler to vtctldgrpcserver etc).

**Comment by frouioui** [comment]
> > Fix for that test is [3567a8c](https://github.com/vitessio/vitess/commit/3567a8cbc5becf0642358a529f380e0bf9309dd1) in case @deepthi thinks it's worth a backport.
> 
> I do, I do 😄 @frouioui and I discussed it and agreed the unit test fix should be back ported.

The backport is available here: https://github.com/vitessio/vitess/pull/9764

**Pull request merged**